addon/components/ui-calendar/template.hbs

```diff
@@ -1,54 +1,55 @@
-<nav local-class="nav">
-  <button 
-    type="button" 
-    local-class="nav-control on-previous-month" 
+<nav local-class="nav" class="ui-calendar-nav">
+  <button
+    type="button"
+    local-class="nav-control on-previous-month" class="ui-calendar-nav-previous"
     {{on "click" (fn this.onMoveCenter -1 "month")}}
   >
-    {{ui-icon "arrow-left-12" size="small"}}
+    {{svg-jar "arrow-left-1" height="15" width="15"}}
```

Comment:
The icon used has been changed from `arrow-left-12` to `arrow-left-1` here - is that right?

Reply:
There are two things here, there was a reference to ui-icon which didn't exist, and arrow-left-12 which also didn't exist. So I changed it to use svg-jar and arrow-left-1

Reply:
Completely understand the UiIcon change. I think this and the svg exist in the parent app I was working in and so didnt spot that they werent available in the addon.

But I just want to compare arrow-left-1 and arrow-left-12 to make sure the change is good from a design POV.

Reply:
I think arrow-left-12 was a typo, it doesn't seem like there is such an svg in the app - https://github.com/goods/ember-assembly/tree/master/public/icons

Reply:
<img width="511" alt="Screenshot 2020-11-26 at 09 52 41" src="https://user-images.githubusercontent.com/38084194/100336633-54f45800-2fce-11eb-8182-79a3a656abab.png">


Hey Kyle, this is how the icons look using the arrow-left-1 and arrow-right-1

Reply:
Hi @beerinho - those arrows seem pretty heavy. Could I see a screenshot of the default ember-assembly calendar without all the 18week theming?

Reply:
For reference here is it with the current arrows

Reply:
<img width="401" alt="Screenshot 2020-11-26 at 11 44 36" src="https://user-images.githubusercontent.com/38084194/100347200-e8348a00-2fdc-11eb-94b4-199c89d7a9b1.png">
arrow-left-1

<img width="401" alt="Screenshot 2020-11-26 at 11 45 01" src="https://user-images.githubusercontent.com/38084194/100347204-e965b700-2fdc-11eb-8b45-9dac76d5fdcd.png">
arrow-left-12

Reply:
![image](https://user-images.githubusercontent.com/371930/100348960-9ccfab00-2fdf-11eb-92b7-b180ad73d187.png)

Reply:
Looks fine with arrow-1. Could you also change the font-weight of the date to 500 please?